lib/samples/create_planar_and_geodetic_buffers/README.md

```diff
@@ -32,6 +32,8 @@ Creating buffers is a core concept in GIS proximity analysis that allows you to
 
 Buffers can be generated as either `planar` (flat - coordinate space of the map's spatial reference) or `geodetic` (technique that considers the curved shape of the Earth's surface, which is generally a more accurate representation). In general, distortion in the map increases as you move away from the standard parallels of the spatial reference's projection. This map is in Web Mercator so areas near the equator are the most accurate. As you move the buffer location north or south from that line, you'll see a greater difference in the polygon size and shape. Planar operations are generally faster, but performance improvement may only be noticeable for large operations (buffering a great number or complex geometry).
 
+Geodetic buffers in the far northern and southern regions of the map will extend beyond the map's limits. The visible extent of the basemap in this sample is limited to between approximately +/-85 degrees latitude while geodetic buffers are calculated to extend all the way to the poles (+/-90 degrees). Also, since map view wrapping is active, geodetic buffers that cross the international date line (180 degrees longitude) will appear with a vertical line on the dateline. Geodetic buffers are generated between +/-180 degrees longitude. Wrapping the map causes the buffer polygons to repeat across the wrapped map. The vertical line denotes the end of one buffer polygon and the beginning of the next. 
```

Comment:
you could mention that the vertical line is a result of the geometry being "normalized", resulting in a multipart geometry - https://developers.arcgis.com/flutter/api-reference/reference/arcgis_maps/GeometryEngine/normalizeCentralMeridian.html

Reply:
Updated description to include normalization.